### ISSUE TITLE
the current train code include gradient accumulation or not?

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-resnet#26]

**Comment by gaopeng-eugene** [comment]
If include, please point out the position in the code.

**Comment by DrSleep** [comment]
No, there is no gradients accumulation at the moment.

**Comment by DrSleep** [comment]
There is gradients accumulation now in the train_msc.py script. Please refer to README for more information
